app/models/profile.rb

```diff
@@ -34,7 +21,23 @@ def activate
     end
   end
 
-  def verified?
-    verified_at.present?
+  def plain_pii
+    @_plain_pii ||= Pii::Attributes.new
```

Comment:
Yup, makes sense!

I would even be in favor of something like this (very verbose and separate but very clear)

```
profile = Profile.new( user: user )
pii_attrs = Pii::Attributes.new_from_hash( ssn: '1234' )
profile.encrypted_pii = pii_attrs.encrypt(passphrase, pii_attrs)
profile.save!

pii_decrypted = Pii::Attributes.decrypt(passphrase, profile.encrypted_pii)
```